### PR TITLE
feat: Replaced listView with recyclerView

### PR DESCRIPTION
**Pull request by code-crusher**
chore: added recyclerView dependency

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -8,12 +7,18 @@
     tools:context=".MainActivity">
 
 
-    <ListView
+    <!--<ListView
```
Comment (the-dagger) [inline app/src/main/res/layout/activity_main.xml]: Why not remove this instead?

Comment (code-crusher) [inline app/src/main/res/layout/activity_main.xml]: @the-dagger Thanks for pointing it out. Done :)

Comment (the-dagger) [inline app/src/main/res/layout/activity_main.xml]: Cool.
Also now onwards, do mention, which Issue a pull request refers to.
For example this PR is for #12

Comment (code-crusher) [inline app/src/main/res/layout/activity_main.xml]: @the-dagger Alright. 👍

**Pull request merged**